pages/borrow/[symbol].tsx

```diff
@@ -154,31 +291,72 @@ const Borrow: NextPage<
                       <div className="flex gap-1">
                         <div className="text-gray-800 dark:text-white">
-                          3.15%
+                          {borrowApr.toFixed(2)}%
                         </div>
                         <div className="text-gray-400">
-                          (10.1234 {asset.underlying.symbol} in interest)
+                          (
+                          {formatUnits(
+                            expectedInterest,
+                            asset.underlying.decimals,
+                            prices[asset.underlying.address] ?? 0,
+                          )}{' '}
+                          {asset.underlying.symbol} in interest)
                         </div>
                       </div>
                     </div>
                     <div className="flex w-full sm:w-fit text-sm gap-2 justify-between">
                       <span className="text-gray-500">LTV</span>
-                      <div className="text-yellow-500">15.24%</div>
+                      <div className="text-yellow-500">{ltv.toFixed(2)}%</div>
                     </div>
                   </div>
                 </div>
                 <button
-                  disabled={true}
-                  className="font-bold text-base sm:text-xl disabled:bg-gray-100 dark:disabled:bg-gray-800 h-12 sm:h-16 rounded-lg disabled:text-gray-300 dark:disabled:text-gray-500"
+                  disabled={
+                    epochs === 0 ||
+                    collateralAmount === 0n ||
+                    loanAmount === 0n ||
+                    collateralAmount > collateralBalance ||
+                    loanAmount > available ||
+                    loanAmount + maxInterest > maxLoanAmountExcludingCouponFee
+                  }
+                  className="font-bold text-base sm:text-xl bg-green-500 disabled:bg-gray-100 dark:disabled:bg-gray-800 h-12 sm:h-16 rounded-lg text-white disabled:text-gray-300 dark:disabled:text-gray-500"
+                  onClick={async () => {
+                    if (!collateral) {
+                      return
+                    }
+                    const hash = await borrow(
+                      collateral,
+                      collateralAmount,
+                      asset,
+                      loanAmount,
+                      epochs,
+                      expectedInterest,
+                    )
+                    if (hash) {
+                      await router.replace('/')
+                    }
+                  }}
                 >
-                  Confirm
+                  {epochs === 0
+                    ? 'Select expiration date'
+                    : collateralAmount === 0n
+                    ? 'Enter collateral amount'
+                    : loanAmount === 0n
+                    ? 'Enter loan amount'
+                    : collateralAmount > collateralBalance
+                    ? 'Not enough collateral'
```

Comment:
This message could be misleading as it sounds like there isn't enough collateral to make the loan.
In cases where the user balance is insufficient, let's use `Insufficient ${symbol} balance` throughout the app.